Cargo.toml

```diff
@@ -70,7 +69,7 @@ shadow-rs = "0.11.0"
 # see: https://github.com/svartalf/rust-battery/issues/33
 starship-battery = { version = "0.7.9", optional = true }
 strsim = "0.10.0"
-sys-info = "0.9.1"
+systemstat = "=0.1.11"
```

Comment:
What does the equal mean within the quotes?
I can't seem to find it in the Cargo.toml reference.

Reply:
The `=` ensures only the specified version is allowed, and newer versions won't be used even without `--locked`.

Reply:
Is there a reason that we're specifically locking this dependency's version?

Reply:
Because it's still at 0.1, there might be API-breaking patch releases.